devices/uart-nrf9160/uart.c

```diff
@@ -0,0 +1,369 @@
+/*
+ * Phoenix-RTOS
+ *
+ * plo - operating system loader
+ *
+ * nRF9160 UART driver
+ *
+ * Copyright 2022 Phoenix Systems
+ * Author: Damian Loewnau
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#include <hal/hal.h>
+#include <lib/errno.h>
+#include <lib/lib.h>
+#include <devices/devs.h>
+
+#include <board_config.h>
+
+
+/* currently supported baud rates - 9600, 115200 */
+#define BAUD_9600_REGVAL   0x00275000u
+#define BAUD_115200_REGVAL 0x01d60000u
```

Comment:
Frankly, these defines are used only in one place so i don't see a need to make them as defines.